doc/reference/views.rst

```diff
@@ -132,20 +149,20 @@ There are three types of element locators for matching a target element:
 The inheritance spec may have an optional ``position`` attribute specifying
 how the matched node should be altered:
 
-``inside`` (default)
+``position="inside"`` (default)
     the content of the inheritance spec is appended to the matched node
-``replace``
+``position="replace"``
     the content of the inheritance spec replaces the matched node.
     Any text node containing only ``$0`` within the contents of the spec will
     be replaced  by a complete copy of the matched node, effectively wrapping
     the matched node.
-``after``
+``position="after"``
     the content of the inheritance spec is added to the matched node's
     parent, after the matched node
-``before``
+``position="before"``
     the content of the inheritance spec is added to the matched node's
     parent, before the matched node
-``attributes``
+``position="attributes"``
```

Comment:
personal opinion, it would be nice to add an example of `position=attributes` usage as it is a bit more tricky than the others